syncd/FlexCounter.cpp

```diff
@@ -1014,6 +1130,18 @@ FlexCounter::FlexCounter(
     m_enable = false;
     m_isDiscarded = false;
 
+    m_configDb = std::make_shared<swss::DBConnector>("CONFIG_DB", 0);
+    m_cfgDeviceMetaDataTable = std::make_shared<swss::Table>(m_configDb.get(), CFG_DEVICE_METADATA_TABLE_NAME);
+
+    getCfgSwitchType(m_switchType);
+
+    if (m_switchType == "voq")
+    {
+       m_appDb = std::make_shared<swss::DBConnector>("CHASSIS_APP_DB", 0, true);
+       m_appDbPipeline = std::make_shared<swss::RedisPipeline>(m_appDb.get());
+       m_appDbCountersTable = std::make_shared<swss::Table>(m_appDbPipeline.get(), "COUNTERS", true);
+    }
+
```

Comment:
all databases names are hardcoded, they should be obrained from defines or config header

Reply:
Same as above